src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -0,0 +1,298 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.CodeAnalysis.Editor.UnitTests;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.LanguageServer;
+using Microsoft.CodeAnalysis.LanguageServer.CustomProtocol;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Text.Adornments;
+using Roslyn.Utilities;
+using Xunit;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Roslyn.Test.Utilities
+{
+    [UseExportProvider]
+    public abstract class AbstractLanguageServerProtocolTests
+    {
+        protected virtual ExportProvider GetExportProvider()
+        {
+            var requestHelperTypes = DesktopTestHelpers.GetAllTypesImplementingGivenInterface(
+                    typeof(IRequestHandler).Assembly, typeof(IRequestHandler));
+            var exportProviderFactory = ExportProviderCache.GetOrCreateExportProviderFactory(
+                TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic
+                .WithPart(typeof(LanguageServerProtocol))
+                .WithParts(requestHelperTypes));
+            return exportProviderFactory.CreateExportProvider();
+        }
+
+        protected static void AssertSymbolInformationsEqual(LSP.SymbolInformation expected, LSP.SymbolInformation actual)
```

Comment:
>Informations [](start = 42, length = 12)

`Information`, not `Informations`

Reply:
Could just be `AssertEqual` overloaded on parameter types.

---
In reply to: [296853199](https://github.com/dotnet/roslyn/pull/36249#discussion_r296853199) [](ancestors = 296853199)